strings/ctype-latin1.c

```diff
@@ -504,19 +504,22 @@ struct charset_info_st my_charset_latin1_nopad=
  *
  * The modern sort order is used, where:
  *
- * '�'  ->  "ae"
- * '�'  ->  "oe"
- * '�'  ->  "ue"
- * '�'  ->  "ss"
+ * 'a' ('a' with diaeresis)  ->  "ae"
+ * 'o' ('o' with diaeresis)  ->  "oe"
+ * 'u' ('u' with diaeresis)  ->  "ue"
+ * 'sharp S'  ->  "ss"
  */
 
 
 /*
  * This is a simple latin1 mapping table, which maps all accented
  * characters to their non-accented equivalents.  Note: in this
- * table, '�' is mapped to 'A', '�' is mapped to 'Y', etc. - all
```

Comment:
Same here. It's sadly much harder to read and I don't think I would easily be able to grok the amended version, had I not had the deleted version for comparison. Same comment for the rest of the file.